docs/changelog/0.5.0.draft.md

```diff
@@ -14,6 +14,7 @@ Check out the documentation at
 * Removed `java.net.URL` related stubs we don't plan to implement.
 
 ## Breaking changes
+* `scala.scalanative.unsafe.Zone.apply` by default uses a context function argument. For Scala 2 cross-compilation use `Zone.acquire` instead. 
```

Comment:
I realize that this .md is draft. When it gets looked at in the
closer to release,  it could stand a reference to the corresponding
section in very well put `Interop.rst`.

Release notes are supposed to be informative, not comprehensive,
but I must confess that I could not follow or implement the change
based on this.

Off topic:
I think a similar "quick reference the appropriate .rst" concept
could also be considered for the "ServiceLoader" section below.

I think these two changes are a little more technical and deeper
that some of the other changes, such as `utsnameOps`.